model/container_registry.py

```diff
@@ -81,6 +95,16 @@ def registry_type(self):
     def api_base_url(self):
         return self.raw.get('api_base_url')
 
+    def rotation_cfg_reference(self) -> model.base.CfgElementReference:
```

Comment:
see comment to aws-cfg.

btw, if you _do_ want do deduplicate this, you might define a "CrossRotationMixin". If going for that approach, it would be fine to standardise on the name.

However, it is perfectly fine (and probably better, as we will have specific code for each element-type anyhow) to keep this type-specific. In which case, however, there should be no hardcoded assumptions as to the attribute-names.